### PR TITLE
Update sample design

### PR DESCRIPTION
**Pull request by TsvetinaIvanova**

### CODE REVIEW

**Comment by didiepetrova** [comment]
When LiveSync is disabled, then pressing the Sync button still starts downloading package and then it suddenly stops, the message that LiveSync is disabled disappears. For the other aspects :+1:

**Comment by TsvetinaIvanova** [comment]
The Enable and Disable button only enable and disable the three-finger gesture LiveSync. Programmatic LiveSync keeps working (got the info from Marto Bektchiev). I updated the enable and disable messages in main.js to make this clear.

File: sample-livesync/scripts/main.js [hunk 1]
```diff
@@ -56,13 +56,13 @@ Application.prototype = {
     enableLiveSync: function () {
         var that = this;
         livesync.enable();
-        that.setResults("LiveSync enabled.");
+        that.setResults("LiveSync with the 3-finger gesture is enabled.");
```
Comment (Mike4o) [inline sample-livesync/scripts/main.js]: three-finger

File: sample-livesync/.abproject [hunk 2]
```diff
@@ -1,67 +1,53 @@
...
+  "projectVersion": 1,
+  "iOSDeviceFamily": [
+    "1",
+    "2"
+  ],
+  "AndroidVersionCode": "1",
 	"AndroidHardwareAcceleration": "true",
-	"CorePlugins": [
+  "CorePlugins": [
 		"cordova-plugin-device",
```
Comment (Mike4o) [inline sample-livesync/.abproject]: Why do we need the Device plugin here?

File: sample-livesync/.abproject [hunk 3]
```diff
@@ -1,67 +1,53 @@
...
 		"cordova-plugin-splashscreen",
 		"cordova-plugin-vibration",
```
Comment (Mike4o) [inline sample-livesync/.abproject]: Why do we need to Vibration plugin? The LiveSync plugin has vibration implemented in it.

**Comment by Mike4o** [comment]
:+1:

**Pull request merged**